src/bin/nydus-image/main.rs

```diff
@@ -356,6 +357,13 @@ fn prepare_cmd_args(bti_string: &'static str) -> App {
                         .action(ArgAction::SetTrue)
                         .required(false)
                 )
+                .arg(
+                    Arg::new("blob-cache-dir")
+                        .long("blob-cache-dir")
+                        .help("Directory path to save generated blob cache files(blob meta and blob data)")
```

Comment:
How about `Directory path to generate blob cache files ($id.blob.meta and $id.blob.data)`.